### PR TITLE
Move common/canvas-* files and update refs

### PR DESCRIPTION
**Pull request by zcorpan**
Also remove common/namespaces.js and fix affected files.

Part of #17913.

### CODE REVIEW

**Review by fserb — APPROVED** [review]

**Comment by foolip** [comment]
This touches a lot of files, can you trigger full runs to gain and compare to master to gain confidence it doesn't break stuff?

**Comment by jugglinmike** [comment]
@zcorpan Here's what @foolip has in mind:

1. Push to the "triggers" branches that we created in gh-18870 for this purpose
2. Wait for the results collection to complete
3. Use the wpt.fyi UI to select the results for your commit (cf7c5bf6c) and compare to the results for the commit on which it's based (819a234d)

I've started things off with the triggers:

    $ git push --force upstream HEAD:triggers/chrome_dev
    $ git push --force upstream HEAD:triggers/firefox_nightly

I believe the results will show up here when they're ready: https://wpt.fyi/runs?sha=cf7c5bf&max-count=100, but you can use the Taskcluster status pages (as displayed in the GitHub commit history for those branches) to monitor the progress

- https://tools.taskcluster.net/task-group-inspector/#/IxItqHUwT0ueOAKVaHpHGw
- https://tools.taskcluster.net/task-group-inspector/#/IxItqHUwT0ueOAKVaHpHGw

Just know that there is some additional delay between Taskcluster's completion and the availability of results on wpt.fyi

**Comment by zcorpan** [comment]
Thanks @jugglinmike !

Diff for Firefox: https://wpt.fyi/results/?diff&filter=ADC&run_id=328910004&run_id=344000007

tl;dr: this change doesn't break stuff.


**Of the changed tests:**

https://wpt.fyi/results/2dcontext/imagebitmap/createImageBitmap-origin.sub.html?diff&filter=ADC&run_id=328910004&run_id=344000007 shows 1 difference in results, but that test appears to be flaky in Firefox according to the [results history](https://wpt.fyi/results/2dcontext/imagebitmap/createImageBitmap-origin.sub.html?label=experimental&label=master&aligned).

**Outside changed tests:**

Some big diffs in css animation tests, but these also seem to be caused by flakyness.
https://wpt.fyi/results/css/css-transforms/animation?diff&filter=ADC&run_id=328910004&run_id=344000007

Since only the canvas tests (in `2dcontext` and `html/semantics/embedded-content/the-canvas-element`) used the `common/canvas-` files (nothing shows up in `git grep "common/canvas-"` on this branch), I assume that any other difference in results are due to flaky tests.
